index.html

```diff
@@ -60,9 +60,13 @@ <h2>
         click, this JavaScript code shares the current page's URL.
       </p>
       <pre class="example javascript" title="Basic usage">
-        shareButton.addEventListener('click', () =&gt; {
-          navigator.share({title: 'Example Page', url: ''})
-              .then(console.log('Share successful'));
+        shareButton.addEventListener("click", async () => {
+          try {
+            await navigator.share({ title: "Example Page", url: "" });
+            console.log("Share successful");
+          } catch (err) {
+            console.error("Oh no! couldn't share.", err.message);
```

Comment:
I don't like this informal message. Can you change the string to "Share failed:"